ui/src/app/components/services/PerformanceService.js

```diff
@@ -18,43 +18,43 @@
             if (performancePeriod === 'week') {
                 return [
                     {
-                        "key": 'Middleware',
-                        "values": [[1, 11], [2, 10], [3, 14], [4, 21], [5, 13], [6, 21], [7, 21], [8, 18], [9, 11], [10, 11], [11, 18], [12, 14], [13, 10], [14, 20], [15, 21], [16, 28], [17, 12], [18, 16], [19, 22], [20, 18], [21, 21], [22, 10], [23, 11], [24, 14], [25, 9], [26, 14], [27, 10], [28, 21], [29, 11]]
+                        'key': 'Middleware',
```

Comment:
This should be double quotes instead of single quotes according to JSON standard 
http://www.json.org/